scripts/add_export.py

```diff
@@ -66,6 +66,9 @@ def select_ports(n):
     hydrogen_buses_ports = n.buses.loc[ports.index + " H2"]
     hydrogen_buses_ports.index.name = "Bus"
 
+    # drop duplicate ports if exists
+    hydrogen_buses_ports = hydrogen_buses_ports.drop_duplicates()
```

Comment:
Thanks a lot @yerbol-akhmetov for this great proposal; I'd propose just a minor revision that may make the code more robust.

If I get the issue right, the problem in the current workflow is that the column "gadm_{}".format(gadm_level) used as index may contain duplicates.
For your case, however, drop_duplicates works.
The function drop_duplicates() by default drops only duplicated rows, if rows have the same index but different content they are not dropped.

To ensure robustness and keep the proposed formulation, a possible proposal would be to move the drop_duplicates few lines up with something like:
``` python
    gcol = "gadm_{}".format(gadm_level)
    ports_sel = ports.loc[~ports[gcol].duplicated(keep='first')].set_index(gcol)
```
May that work? Feel free to revise and improve as you feel best

I'd also recommend to add a comment like "TODO: revise if ports quantity and property by shape become relevant" just to keep track of this approximation

Reply:
Thanks @davide-f, @ekatef. I have applied the latest changes that were suggested and tested it locally. It worked as intended. Also, the comment about TODO was added too.

Reply:
Ahh, true!! The check for duplicates relate to the whole lines, not only indices. Agree then that data loss is highly unlikely.

I think the implementation looks great, and if @davide-f doesn't have further comments, happy to merge 😄